autospec/specfiles.py

```diff
@@ -1586,6 +1586,19 @@ def write_ant_pattern(self):
         self.write_install_prepend()
         self._write_strip("")
 
+    def write_gradle_pattern(self):
+        self.write_prep()
+        self._write_strip("%build")
+        self.write_build_prepend()
+        self.write_proxy_exports()
+        self._write_strip("mkdir -p /builddir/.m2")
+        self._write_strip("cp -r /usr/share/java/.m2/* /builddir/.m2/")
+        self._write_strip("gradle --offline " + self.extra_make)
```

Comment:
Did you find a way to get gradle to use Maven's local repo by default? So far I only came up with this:
`sed -i 's/\(repositories\s*{\)/\1\n    mavenLocal()/' build.gradle`

Reply:
Sadly, no. Now I'm reading the documentation for newer releases of gradle to see if there is an improvement we could use. The offline flag only search on the gradle cache for the missing stuff.